src/main/java/uk/ac/ebi/subs/agent/converters/UsiAttributeToBsdAttribute.java

```diff
@@ -19,16 +23,21 @@
 
     @Override
     public Attribute convert(uk.ac.ebi.subs.data.component.Attribute usiAttribute) {
-        String uri = null;
-        if(usiAttribute.getTerms() != null && !usiAttribute.getTerms().isEmpty()) {
-            uri = usiAttribute.getTerms().get(0).getUrl();
 
+        Collection<String> termUrls = Collections.emptySet();
+
+
+        if(usiAttribute.getTerms() != null) {
+            termUrls = usiAttribute.getTerms().stream()
+                    .map(Term::getUrl)
+                    .filter(term -> term != null)
```

Comment:
Can we name it uri/url/iri instead of term, please?

Reply:
Yes